local-builds.gradle-example

```diff
@@ -16,4 +16,5 @@ binary build if you're not working on a specific library.
 ext {
     localFluxCPath = "../WordPress-FluxC-Android"
     localWPUtilsPath = "../WordPress-Utils-Android"
+    localGutenbergMobilePath = "../gutenberg-mobile"
```

Comment:
Would it make more sense to comment all these out by default? The odds that someone would have all three of these setup at the proper path and actually want to build all three of these from source seems to be virtually 0, so it seems like a non-ideal starting point. It also virtually guarantees that the first build after copying this file will fail. The failure is relatively clear, but it does present a stumbling block, particularly if someone copies the file over not fully understanding what it is doing. Just a thought.

Reply:
Whenever this file is copied, I expect at least one of the libraries to be used as a composite build, so I thought the developer would decide which ones to comment out and which ones to keep. Having said that, commenting out them sounds like a reasonable default. Updated in cc4f290b63a378f6773db74370ec03796426a594.